modules/customizing-the-appearance/proc-overriding-translations.adoc

```diff
@@ -0,0 +1,85 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="prov-overriding-translations_{context}"]
+= Overriding translations
+In {product-very-short} 1.8, English and French are the supported languages. You can implement transalations for other languages by using an `allTranslations.json` file to override the existing translation strings, and updating the `i18n` section of your `{my-app-config-file}` configuration file to include the translation override file.
+
+.Prerequisitea
+* You have enabled localization in your {product-very-short} application.
+
+.Procedure
+. In the top user menu, go to *Settings* > *General*.
+. Click on the download link in the *Translations* panel to download the default English translation strings.
```

Comment:
This is no longer part of the 1.8 release
Sorry for the confusion

You can find more details in the comments section here https://issues.redhat.com/browse/RHIDP-8892